study.md

```diff
@@ -43,5 +47,6 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash URLs separate the server side URL portion of the URL from the
+portion that Ember uses.
```

Comment:
This is important when using GitHub Pages because GitHub doesn't have an Ember server running in the background, serving up pages; instead, the server is GHPages' own static page server. Using a Hash URL allows GH Pages to load up the root page, and to handle routing by running Ember as Javascript for that page.